Elizabeth_Theme/src/js/main.js

```diff
@@ -0,0 +1,3 @@
+window.addEventListener('scroll', () => {
+	document.querySelector('header').classList.toggle('scrolled', document.documentElement.scrollTop>150);
```

Comment:
Missing space before and after `>`.

Reply:
Before using a DOM node in JS, we should check to make sure the DOM node exists.